examples/bridge-app/esp32/README.md

```diff
@@ -242,6 +242,6 @@ Parameters:
 To use the Client to send Matter commands, run the built executable and pass it
 the target cluster name, the target command name as well as an endpoint id.
 
-    $ .out/debug/chip-tool onoff on 12344321 1
+    $ ./out/debug/chip-tool onoff on 12344321 1
```

Comment:
This README will be updated soon. For now, please replace the endpoint 1 to 2 in the onoff command as the light isn't bridged on endpoint 1

Reply:
Done